src/ir/context.rs

```diff
@@ -2318,6 +2319,21 @@ If you encounter an error missing from this list, please file an issue or a PR!"
                         return true;
                     }
 
+                    // Items with a source location in an explicitly allowlisted file
+                    // are always included.
+                    if let Some(location) = &item.location() {
+                        let (file, _, _, _) = location.location();
+                        if let Some(filename) = file.name() {
```

Comment:
This does a fair amount of ffi calls and allocations, so let's check that `allowlisted_files` has something before bothering getting the filename.

Reply:
Done. (There's a check for any kind of allowlist further up that I thought would be sufficient.)